actions/approve_csr_handler.go

```diff
@@ -30,13 +31,17 @@ type approveCSRHandler struct {
 	csrFetchInterval       time.Duration
 }
 
-func (h *approveCSRHandler) Handle(ctx context.Context, data interface{}) error {
+func (h *approveCSRHandler) Handle(ctx context.Context, data interface{}, actionID string) error {
```

Comment:
maybe instead of passing actionID, we should just pass whole action and then use `action.Data()` for actual action data